xcp_d/utils/dcan2fmriprep.py

```diff
@@ -11,315 +12,411 @@
-        for ttt in taskid:
-            taskdir = "task-" + ttt
-
-            taskname = re.split(r"(\d+)", ttt)[0]
-            run_id = "_run-" + str(int(re.split(r"(\d+)", ttt)[1]))
-            func_dirxx = func_dirx + taskdir
-
-            sbref = func_dirxx + "/" + taskdir + "_SBRef.nii.gz"
-            volume = func_dirxx + "/" + taskdir + ".nii.gz"
-
-            brainmask = func_dirxx + "/brainmask_fs.2.0.nii.gz"
-            dtsereis = func_dirxx + "/" + taskdir + "_Atlas.dtseries.nii"
-            motionp = func_dirxx + "/Movement_Regressors.txt"
-            rmsdx = func_dirxx + "/Movement_AbsoluteRMS.txt"
-
-            mvreg = pd.read_csv(motionp, header=None, delimiter=r"\s+")
-            mvreg = mvreg.iloc[:, 0:6]
-            mvreg.columns = ["trans_x", "trans_y", "trans_z", "rot_x", "rot_y", "rot_z"]
-            # convert rot to rad
-            mvreg["rot_x"] = mvreg["rot_x"] * np.pi / 180
-            mvreg["rot_y"] = mvreg["rot_y"] * np.pi / 180
-            mvreg["rot_z"] = mvreg["rot_z"] * np.pi / 180
+        fsaverage_dir_orig = os.path.join(anat_dir_orig, "fsaverage_LR32k")
 
-            csfreg = extractreg(mask=csfmask, nifti=volume)
-            wmreg = extractreg(mask=wmmask, nifti=volume)
-            gsreg = extractreg(mask=brainmask, nifti=volume)
-            rsmd = np.loadtxt(rmsdx)
+        # NOTE: Why glob? Do we not know the full filenames? Are there multiple files?
+        rh_midthickness_orig = glob.glob(
+            os.path.join(fsaverage_dir_orig, "*R.midthickness.32k_fs_LR.surf.gii")
+        )[0]
+        rh_midthickness_fmriprep = os.path.join(
+            anat_dir_fmriprep,
+            f"{sub_id}_{ses_id}_space-fsLR_den-32k_hemi-R_desc-hcp_midthickness.surf.gii",
+        )
+        copy_dictionary[rh_midthickness_orig] = [rh_midthickness_fmriprep]
+
+        lh_midthickness_orig = glob.glob(
+            os.path.join(fsaverage_dir_orig, "*L.midthickness.32k_fs_LR.surf.gii")
+        )[0]
+        lh_midthickness_fmriprep = os.path.join(
+            anat_dir_fmriprep,
+            f"{sub_id}_{ses_id}_space-fsLR_den-32k_hemi-L_desc-hcp_midthickness.surf.gii",
+        )
+        copy_dictionary[lh_midthickness_orig] = [lh_midthickness_fmriprep]
+
+        rh_inflated_orig = glob.glob(
+            os.path.join(fsaverage_dir_orig, "*R.inflated.32k_fs_LR.surf.gii")
+        )[0]
+        rh_inflated_fmriprep = os.path.join(
+            anat_dir_fmriprep,
+            f"{sub_id}_{ses_id}_space-fsLR_den-32k_hemi-R_desc-hcp_inflated.surf.gii",
+        )
+        copy_dictionary[rh_inflated_orig] = [rh_inflated_fmriprep]
+
+        lh_inflated_orig = glob.glob(
+            os.path.join(fsaverage_dir_orig, "*L.inflated.32k_fs_LR.surf.gii")
+        )[0]
+        lh_inflated_fmriprep = os.path.join(
+            anat_dir_fmriprep,
+            f"{sub_id}_{ses_id}_space-fsLR_den-32k_hemi-L_desc-hcp_inflated.surf.gii",
+        )
+        copy_dictionary[lh_inflated_orig] = [lh_inflated_fmriprep]
```

Comment:
@madisoth mentioned that the DCAN and HCP pipelines should output the HCP-style very-inflated surfaces too. I'd like to make sure we grab those as part of this refactor.

Reply:
The very-inflated surfaces are named `very_inflated`.